x-pack/plugins/fleet/server/services/epm/registry/index.ts

```diff
@@ -74,64 +75,73 @@ async function _fetchFindLatestPackage(
   packageName: string,
   options?: FetchFindLatestPackageOptions
 ) {
+  const logger = appContextService.getLogger();
```

Comment:
These fetch functions could really use some unit tests at this point now that there are varying fallback cases, etc.

Reply:
Good idea, I added an item in our board 👍